netbout-web/src/test/java/com/netbout/rest/bout/XeMessageTest.java

```diff
@@ -70,6 +69,7 @@ public String author() {
                     }
                 }
             ).toXembly()
+        // @checkstyle IndentationCheck (1 line)
```

Comment:
@mkordas Same problem - '.' have incorrect indentation level 8, expected level should be 12. (IndentationCheck)